doc/api/tracing.md

```diff
@@ -1,12 +1,14 @@
-# Tracing
+# Trace Events
 
 <!--introduced_in=v7.7.0-->
 
+> Stability: 1 - Experimental
+
 Trace Event provides a mechanism to centralize tracing information generated by
 V8, Node.js core, and userspace code.
 
-Tracing can be enabled by passing the `--trace-events-enabled` flag when
-starting a Node.js application.
+Tracing can be enabled with the `--trace-event-categories` command-line flag
+or by using the `trace_events` module.
 
 The set of categories for which traces are recorded can be specified using the
```

Comment:
How come? They seem useful.

Reply:
I agree with @ofrobots and think the current formulation as separate paragraphs is confusing. They are written as if they are about two different flags but they're about the same flag. Put the information in one place.

Reply:
Also, the way @ofrobots re-wrote the second sentence (to be less wordy and to eliminate the passive voice) is much more readable IMO.

Reply:
(Although get rid of the `accepts accepts` in the second sentence. One `accepts` is enough. 😄)

Reply:
(Also: `comma separated` -> `comma-separated`.)

Reply:
If you want to keep them  as separate paragraphs on the grounds that one is about enabling the feature and the other is about specifying categories, rewrite the second one so that it doesn't sound like it's introducing a new flag when it's not. Maybe this?

> Categories for which traces are recorded are specified as one or more names after the `--trace-event-categories` flag.